charts/polkadot-registrar-challenger/values.yaml

```diff
@@ -2,17 +2,19 @@ environment: production
 
 image:
   repo: web3f/polkadot-registrar-challenger
-  tag: v0.1.1
+  tag: v0.1.2
 
 config:
-  port: 3000  
-
-env:
-  matrix:
-    homeServer: 'https://matrix-client.matrix.org'
-    user: 'test-account'
-    pass: 'xxxxx'     
-
+  port: 3000
```

Comment:
this will be the port where the http server will host the healthcheck endpoint as soon it is ready